plugins/civmodcore-paper/src/main/java/vg/civcraft/mc/civmodcore/commands/ChunkMetaCommand.java

```diff
@@ -0,0 +1,33 @@
+package vg.civcraft.mc.civmodcore.commands;
+
+import co.aikar.commands.BaseCommand;
+import co.aikar.commands.annotation.CommandAlias;
+import co.aikar.commands.annotation.CommandPermission;
+import co.aikar.commands.annotation.Description;
+import co.aikar.commands.annotation.Subcommand;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.NamedTextColor;
+import org.bukkit.command.CommandSender;
+
+@CommandAlias("cmc")
+public class ChunkMetaCommand extends BaseCommand {
+
+	private static boolean ENABLE_CHUNK_META_LOGS = false;
```

Comment:
The command shouldn't hold what's effectively global state itself, there should be some dedicated collection of feature flags be it the config, or secondarily derived from the config

Reply:
No it shouldn't but it was the easiest option for a very accessory feature